app/stable_appconfig.go

```diff
@@ -49,60 +44,23 @@ func setCustomKVStoreKeys() []string {
 }
 
 func (app *RegenApp) registerUpgradeHandlers() {
+
 	// This is the upgrade plan name we used in the gov proposal.
-	upgradeName := "v2.0-upgrade"
-	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
```

Comment:
We can also remove IBC patch [handler](https://github.com/regen-network/regen-ledger/blob/v3.0.0-rc1/app/app.go#L506-L509).

Reply:
true, thanks for note